playground/src/pages/CodingEnvironment.tsx

```diff
@@ -283,7 +283,7 @@ export default function CodingEnvironment(): JSX.Element {
       if (inputObject.advice_stack) {
         setAdviceValue(formatBeautifyNumbersArray(inputObject.advice_stack));
         setIsAdviceStackLayoutVisible(true);
-      }
+      } // eslint-disable-next-line
     } catch (error: any) {
```

Comment:
Can you simply move line 288 up to line 287?

```suggestion
    } catch (error: any) { // eslint-disable-line @typescript-eslint/no-explicit-any
```

This should work.